vsphere/datadog_checks/vsphere/__about__.py

```diff
@@ -1,4 +1,4 @@
-# (C) Datadog, Inc. 2018
+# (C) Datadog, Inc. 2018-2019
```

Comment:
May need to rebase once #5384 is merged.

Reply:
Hooray 😄